hub/ui/src/components/task/Task.tsx

```diff
@@ -118,6 +121,23 @@ const Task: React.FC<TaskProp> = (props: any) => {
       return str.toUpperCase();
     }));
   }
+  // resource summary
+  let resourceSummary = '';
+  const summaryLength = props.task.description.length;
+  const summaryIndex = props.task.description.indexOf('\n');
+  if (summaryLength > 120) {
```

Comment:
Do we need the variable `summaryLength` instead can't we just directly use `props.task.description.length`

Reply:
yeah,I did it intentionally to avoid many time computation of props.task.description.length

Reply:
have removed variable I think computation would not be much overhead